### PR TITLE
data: vaibhav-xt

### PR DESCRIPTION
**Pull request by vaibhav-xt**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6264"></a>

### CODE REVIEW

File: data/vaibhav-xt.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Vaibhav Maurya",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | Web3.0 Enthusiast | Content Creator | Teaching | Open 😺 Source Contributor | Gaming",
+  "avatar": "https://github.com/vaibhav-xt.png"
```
Comment (shyamtawli) [inline data/vaibhav-xt.json:6]:
```suggestion
  "avatar": "https://github.com/vaibhav-xt.png",
```

File: data/vaibhav-xt.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/vaibhavmaurya09/",
+      "icon": "SiLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/vaibhav-xt",
+      "icon": "FaGithub"
+    },
```
Comment (shyamtawli) [inline data/vaibhav-xt.json:43]:
```suggestion
    }
```

**Review by shyamtawli — CHANGES_REQUESTED** [review]
Welcome to EddieHub @vaibhav-xt, Please make changes as mentioned above.

File: data/vaibhav-xt.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "Node.js",
+    "Python",
+    "Blockchain"
+  ],
+  "socials": [
+    { "icon": "SiYoutube", "url": "https://www.youtube.com/@vaibhav-xt" },
+    { "icon": "SiDiscord", "url": "https://www.youtube.com/@vaibhav-xt" },
+    { "icon": "SiTwitter", "url": "https://twitter.com/vaibhav_txt" },
+    { "icon": "SiGithub", "url": "https://github.com/vaibhav-xt" }
+  ],
```
Comment (shyamtawli) [inline data/vaibhav-xt.json:59]:
```suggestion
  ]
```
remove this also

**Review by shyamtawli — CHANGES_REQUESTED** [review]
remove this `,` also

File: data/vaibhav-xt.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Vaibhav Maurya",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/vaibhav-xt.json:3]: this is no longer needed:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
look good

**Pull request merged**